opm/parser/eclipse/EclipseState/SummaryConfig/tests/SummaryConfigTests.cpp

```diff
@@ -82,6 +85,21 @@ static std::vector< std::string > sorted_keywords( const SummaryConfig& summary
     return ret;
 }
 
+static std::vector< std::string > sorted_key_names( const SummaryConfig& summary ) {
+    std::vector< std::string > ret;
+    for( const auto& x : summary ) {
+        std::string key = x.keyword();
```

Comment:
This can be simplified to: 
```C++
ret.push_back( x.key1() );
```

Where the `key1()` method is the correct composite key.